scripts/run_kickstart_tests.sh

```diff
@@ -296,11 +296,10 @@ for t in ${tests}; do
         ks=${t/.sh/.ks.in}
     fi
     tests_ks+="${ks/.ks.in/.ks} "
```

Comment:
Can we get rid of tests_ks completely? I don't see it used elsewhere.

Reply:
Good point, thanks! I'll do that, but I'll first let @M4rtinK review this in case he has more fundamental objections.

Reply:
Done (it was quick, and with that we already get a new test run)